RuntimeAttachTest fails locally

## Test method
when I run tests with `./mvn clean test` - the test `co.elastic.apm.testapp.RuntimeAttachTest` always falls into error

## Test logs

<details>
<summary>Click to expand</summary>

```
[INFO] -------------------< co.elastic.apm:runtime-attach >--------------------
[INFO] Building co.elastic.apm:runtime-attach 1.26.1-SNAPSHOT         [125/127]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-java) @ runtime-attach ---
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:prepare-agent (default) @ runtime-attach ---
[INFO] argLine set to -javaagent:/home/kananindzya/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/target/jacoco.exec,excludes=**/*.esclazz
[INFO] 
[INFO] --- license-maven-plugin:4.1:format (update-license-headers) @ runtime-attach ---
[INFO] Updating license headers...
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:report (report) @ runtime-attach ---
[INFO] Skipping JaCoCo execution due to missing execution data file.
[INFO] 
[INFO] -----------------< co.elastic.apm:runtime-attach-app >------------------
[INFO] Building co.elastic.apm:runtime-attach-app 1.26.1-SNAPSHOT     [126/127]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-java) @ runtime-attach-app ---
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:prepare-agent (default) @ runtime-attach-app ---
[INFO] argLine set to -javaagent:/home/kananindzya/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-app/target/jacoco.exec,excludes=**/*.esclazz
[INFO] 
[INFO] --- license-maven-plugin:4.1:format (update-license-headers) @ runtime-attach-app ---
[INFO] Updating license headers...
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ runtime-attach-app ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ runtime-attach-app ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to /home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-app/target/classes
[WARNING] [options] bootstrap class path not set in conjunction with -source 7
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ runtime-attach-app ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-app/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ runtime-attach-app ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ runtime-attach-app ---
[INFO] No tests to run.
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:report (report) @ runtime-attach-app ---
[INFO] Skipping JaCoCo execution due to missing execution data file.
[INFO] 
[INFO] -----------------< co.elastic.apm:runtime-attach-test >-----------------
[INFO] Building co.elastic.apm:runtime-attach-test 1.26.1-SNAPSHOT    [127/127]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-java) @ runtime-attach-test ---
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:prepare-agent (default) @ runtime-attach-test ---
[INFO] argLine set to -javaagent:/home/kananindzya/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-test/target/jacoco.exec,excludes=**/*.esclazz
[INFO] 
[INFO] --- license-maven-plugin:4.1:format (update-license-headers) @ runtime-attach-test ---
[INFO] Updating license headers...
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ runtime-attach-test ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-test/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ runtime-attach-test ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ runtime-attach-test ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-test/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ runtime-attach-test ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-test/target/test-classes
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ runtime-attach-test ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running co.elastic.apm.testapp.RuntimeAttachTest
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.078 s <<< FAILURE! - in co.elastic.apm.testapp.RuntimeAttachTest
[ERROR] selfAttach  Time elapsed: 0.054 s  <<< ERROR!
java.lang.IllegalStateException: Unable to find packaged test application in folder :/home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-app/target
        at co.elastic.apm.testapp.RuntimeAttachTest.lambda$findJar$7(RuntimeAttachTest.java:277)
        at java.base/java.util.Optional.orElseThrow(Optional.java:408)
        at co.elastic.apm.testapp.RuntimeAttachTest.findJar(RuntimeAttachTest.java:276)
        at co.elastic.apm.testapp.RuntimeAttachTest.getClassJarLocation(RuntimeAttachTest.java:221)
        at co.elastic.apm.testapp.RuntimeAttachTest.getAppJar(RuntimeAttachTest.java:196)
        at co.elastic.apm.testapp.RuntimeAttachTest.startAppForkedJvm(RuntimeAttachTest.java:188)
        at co.elastic.apm.testapp.RuntimeAttachTest.selfAttach(RuntimeAttachTest.java:118)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
        at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
        at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
        at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

[ERROR] cliAttach  Time elapsed: 0.003 s  <<< ERROR!
java.lang.IllegalStateException: Unable to find packaged test application in folder :/home/kananindzya/Documents/OpenSource/apm-agent-java/integration-tests/runtime-attach/runtime-attach-app/target
        at co.elastic.apm.testapp.RuntimeAttachTest.lambda$findJar$7(RuntimeAttachTest.java:277)
        at java.base/java.util.Optional.orElseThrow(Optional.java:408)
        at co.elastic.apm.testapp.RuntimeAttachTest.findJar(RuntimeAttachTest.java:276)
        at co.elastic.apm.testapp.RuntimeAttachTest.getClassJarLocation(RuntimeAttachTest.java:221)
        at co.elastic.apm.testapp.RuntimeAttachTest.getAppJar(RuntimeAttachTest.java:196)
        at co.elastic.apm.testapp.RuntimeAttachTest.startAppForkedJvm(RuntimeAttachTest.java:188)
        at co.elastic.apm.testapp.RuntimeAttachTest.cliAttach(RuntimeAttachTest.java:95)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
        at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
        at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
        at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   RuntimeAttachTest.cliAttach:95->startAppForkedJvm:188->getAppJar:196->getClassJarLocation:221->findJar:276->lambda$findJar$7:277 IllegalState
[ERROR]   RuntimeAttachTest.selfAttach:118->startAppForkedJvm:188->getAppJar:196->getClassJarLocation:221->findJar:276->lambda$findJar$7:277 IllegalState
[INFO] 
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for co.elastic.apm:apm-agent-parent 1.26.1-SNAPSHOT:
[INFO] 
[INFO] co.elastic.apm:apm-agent-parent .................... SUCCESS [  1.228 s]
[INFO] co.elastic.apm:apm-agent-plugin-sdk ................ SUCCESS [  2.704 s]
[INFO] co.elastic.apm:apm-agent-common .................... SUCCESS [  3.471 s]
[INFO] co.elastic.apm:apm-agent-cached-lookup-key ......... SUCCESS [  0.660 s]
[INFO] co.elastic.apm:apm-indy-bootstrap-module ........... SUCCESS [  0.036 s]
[INFO] co.elastic.apm:apm-agent-core ...................... SUCCESS [02:27 min]
[INFO] co.elastic.apm:apm-agent-plugins ................... SUCCESS [  0.059 s]
[INFO] co.elastic.apm:apm-httpclient-core ................. SUCCESS [  8.366 s]
[INFO] co.elastic.apm:apm-apache-httpclient-plugin ........ SUCCESS [ 18.759 s]
[INFO] co.elastic.apm:apm-agent-api ....................... SUCCESS [  0.228 s]
[INFO] co.elastic.apm:apm-api-plugin ...................... SUCCESS [ 30.243 s]
[INFO] co.elastic.apm:apm-asynchttpclient-plugin .......... SUCCESS [ 27.244 s]
[INFO] co.elastic.apm:apm-jdbc-plugin ..................... SUCCESS [ 15.336 s]
[INFO] co.elastic.apm:apm-java-concurrent-plugin .......... SUCCESS [ 46.867 s]
[INFO] co.elastic.apm:apm-cassandra ....................... SUCCESS [  0.017 s]
[INFO] co.elastic.apm:apm-cassandra-core-plugin ........... SUCCESS [  0.048 s]
[INFO] co.elastic.apm:apm-cassandra3-plugin ............... SUCCESS [  0.259 s]
[INFO] co.elastic.apm:apm-cassandra4-plugin ............... SUCCESS [  0.454 s]
[INFO] co.elastic.apm:apm-okhttp-plugin ................... SUCCESS [ 19.023 s]
[INFO] co.elastic.apm:apm-dubbo-plugin .................... SUCCESS [ 31.384 s]
[INFO] co.elastic.apm:apm-error-logging-plugin ............ SUCCESS [ 10.778 s]
[INFO] co.elastic.apm:apm-es-restclient-plugin ............ SUCCESS [  0.019 s]
[INFO] co.elastic.apm:apm-es-restclient-plugin-common ..... SUCCESS [  1.737 s]
[INFO] co.elastic.apm:apm-es-restclient-plugin-5_6 ........ SUCCESS [  0.355 s]
[INFO] co.elastic.apm:apm-es-restclient-plugin-6_4 ........ SUCCESS [  2.271 s]
[INFO] co.elastic.apm:apm-grails-plugin ................... SUCCESS [ 10.049 s]
[INFO] co.elastic.apm:apm-grpc ............................ SUCCESS [  0.224 s]
[INFO] co.elastic.apm:apm-grpc-plugin ..................... SUCCESS [ 10.993 s]
[INFO] co.elastic.apm:apm-hibernate-search-plugin ......... SUCCESS [  0.013 s]
[INFO] co.elastic.apm:apm-hibernate-search-plugin-common .. SUCCESS [  0.182 s]
[INFO] co.elastic.apm:apm-hibernate-search-plugin-5_x ..... SUCCESS [ 12.656 s]
[INFO] co.elastic.apm:apm-hibernate-search-plugin-6_x ..... SUCCESS [ 14.042 s]
[INFO] co.elastic.apm:apm-servlet-plugin .................. SUCCESS [ 49.665 s]
[INFO] co.elastic.apm:apm-javalin-plugin .................. SUCCESS [ 16.022 s]
[INFO] co.elastic.apm:apm-jaxrs-plugin .................... SUCCESS [ 29.939 s]
[INFO] co.elastic.apm:apm-jaxws-plugin .................... SUCCESS [  6.805 s]
[INFO] co.elastic.apm:apm-jdk-httpclient-plugin ........... SUCCESS [ 10.578 s]
[INFO] co.elastic.apm:apm-jdk-httpserver-plugin ........... SUCCESS [  7.545 s]
[INFO] co.elastic.apm:apm-redis-plugin .................... SUCCESS [  0.010 s]
[INFO] co.elastic.apm:apm-redis-common .................... SUCCESS [  1.347 s]
[INFO] co.elastic.apm:apm-jedis-plugin .................... SUCCESS [  7.328 s]
[INFO] co.elastic.apm:apm-jms-plugin ...................... SUCCESS [  0.048 s]
[INFO] co.elastic.apm:apm-jms-plugin-base ................. SUCCESS [  3.494 s]
[INFO] co.elastic.apm:apm-jmx-plugin ...................... SUCCESS [  4.516 s]
[INFO] co.elastic.apm:apm-jsf-plugin ...................... SUCCESS [  0.053 s]
[INFO] co.elastic.apm:apm-kafka-plugin .................... SUCCESS [  0.008 s]
[INFO] co.elastic.apm:apm-kafka-base-plugin ............... SUCCESS [  0.208 s]
[INFO] co.elastic.apm:apm-kafka-headers-plugin ............ SUCCESS [  0.317 s]
[INFO] co.elastic.apm:apm-lettuce-plugin .................. SUCCESS [ 12.349 s]
[INFO] co.elastic.apm:apm-log-correlation-plugin .......... SUCCESS [  6.956 s]
[INFO] co.elastic.apm:apm-log-shipper-plugin .............. SUCCESS [  6.368 s]
[INFO] co.elastic.apm:apm-log-shader-plugin ............... SUCCESS [  0.012 s]
[INFO] co.elastic.apm:apm-log-shader-plugin-common ........ SUCCESS [  5.427 s]
[INFO] co.elastic.apm:apm-log4j1-plugin ................... SUCCESS [  6.638 s]
[INFO] co.elastic.apm:apm-log4j2-plugin ................... SUCCESS [  8.768 s]
[INFO] co.elastic.apm:apm-logback-plugin .................. SUCCESS [  0.009 s]
[INFO] co.elastic.apm:apm-logback-plugin-impl ............. SUCCESS [ 10.093 s]
[INFO] co.elastic.apm:apm-micrometer-plugin ............... SUCCESS [  8.643 s]
[INFO] co.elastic.apm:apm-mongoclient-plugin .............. SUCCESS [  1.566 s]
[INFO] co.elastic.apm:apm-opentracing-plugin .............. SUCCESS [  0.078 s]
[INFO] co.elastic.apm:apm-process-plugin .................. SUCCESS [ 10.823 s]
[INFO] co.elastic.apm:apm-profiling-plugin ................ SUCCESS [ 52.866 s]
[INFO] co.elastic.apm:apm-quartz-job-plugin ............... SUCCESS [  8.040 s]
[INFO] co.elastic.apm:apm-rabbitmq ........................ SUCCESS [  0.015 s]
[INFO] co.elastic.apm:apm-rabbitmq-plugin ................. SUCCESS [  2.986 s]
[INFO] co.elastic.apm:apm-rabbitmq-spring ................. SUCCESS [01:30 min]
[INFO] co.elastic.apm:apm-reactor-plugin .................. SUCCESS [  9.453 s]
[INFO] co.elastic.apm:apm-redisson-plugin ................. SUCCESS [ 11.084 s]
[INFO] co.elastic.apm:apm-scala-concurrent-plugin ......... SUCCESS [ 28.643 s]
[INFO] co.elastic.apm:apm-scheduled-annotation-plugin ..... SUCCESS [  8.229 s]
[INFO] co.elastic.apm:apm-sparkjava-plugin ................ SUCCESS [  9.930 s]
[INFO] co.elastic.apm:apm-spring-resttemplate ............. SUCCESS [  0.014 s]
[INFO] co.elastic.apm:apm-spring-resttemplate-plugin ...... SUCCESS [ 13.866 s]
[INFO] co.elastic.apm:apm-spring-webflux .................. SUCCESS [  0.010 s]
[INFO] co.elastic.apm:apm-spring-webflux-testapp .......... SUCCESS [  1.892 s]
[INFO] co.elastic.apm:apm-spring-webflux-plugin ........... SUCCESS [01:12 min]
[INFO] co.elastic.apm:apm-spring-webmvc-plugin ............ SUCCESS [ 39.304 s]
[INFO] co.elastic.apm:apm-struts-plugin ................... SUCCESS [ 10.213 s]
[INFO] co.elastic.apm:apm-urlconnection-plugin ............ SUCCESS [ 11.504 s]
[INFO] co.elastic.apm:apm-vertx ........................... SUCCESS [  0.130 s]
[INFO] co.elastic.apm:apm-vertx-common .................... SUCCESS [  1.745 s]
[INFO] co.elastic.apm:apm-vertx3-plugin ................... SUCCESS [ 32.979 s]
[INFO] co.elastic.apm:apm-vertx4-plugin ................... SUCCESS [ 38.877 s]
[INFO] co.elastic.apm:apm-agent ........................... SUCCESS [  6.290 s]
[INFO] co.elastic.apm:elastic-apm-agent ................... SUCCESS [  2.752 s]
[INFO] co.elastic.apm:apm-agent-benchmarks ................ SUCCESS [  0.759 s]
[INFO] co.elastic.apm:apm-spring-resttemplate-test ........ SUCCESS [  0.092 s]
[INFO] co.elastic.apm:apm-logback-plugin-legacy-tests ..... SUCCESS [  6.384 s]
[INFO] co.elastic.apm:apm-es-restclient-plugin-7_x ........ SUCCESS [  0.266 s]
[INFO] co.elastic.apm:apm-jms-spring-plugin ............... SUCCESS [ 13.990 s]
[INFO] co.elastic.apm:apm-jedis-2-tests ................... SUCCESS [  8.920 s]
[INFO] co.elastic.apm:apm-jedis-3-tests ................... SUCCESS [  9.335 s]
[INFO] co.elastic.apm:apm-lettuce-3-tests ................. SUCCESS [  7.774 s]
[INFO] co.elastic.apm:apm-grpc-test-1.6.1 ................. SUCCESS [ 26.805 s]
[INFO] co.elastic.apm:apm-grpc-test-latest ................ SUCCESS [ 32.635 s]
[INFO] co.elastic.apm:apm-rabbitmq-test-3 ................. SUCCESS [  0.080 s]
[INFO] co.elastic.apm:apm-rabbitmq-test-4 ................. SUCCESS [  0.075 s]
[INFO] co.elastic.apm:apm-okhttp-test ..................... SUCCESS [  0.070 s]
[INFO] co.elastic.apm:apm-vertx3-test-latest .............. SUCCESS [ 32.435 s]
[INFO] co.elastic.apm:apm-servlet-jakarta-test ............ SUCCESS [ 17.067 s]
[INFO] co.elastic.apm:apm-opentracing ..................... SUCCESS [ 11.351 s]
[INFO] co.elastic.apm:integration-tests ................... SUCCESS [  0.008 s]
[INFO] co.elastic.apm:simple-webapp ....................... SUCCESS [  0.051 s]
[INFO] co.elastic.apm:jakartaee-simple-webapp ............. SUCCESS [  0.050 s]
[INFO] co.elastic.apm:jsf-app ............................. SUCCESS [  0.006 s]
[INFO] co.elastic.apm:jsf-app-dependent ................... SUCCESS [  0.012 s]
[INFO] co.elastic.apm:jsf-app-standalone .................. SUCCESS [  0.017 s]
[INFO] co.elastic.apm:cdi-app ............................. SUCCESS [  0.006 s]
[INFO] co.elastic.apm:cdi-app-dependent ................... SUCCESS [  0.013 s]
[INFO] co.elastic.apm:cdi-app-standalone .................. SUCCESS [  0.015 s]
[INFO] co.elastic.apm:soap-test ........................... SUCCESS [  0.038 s]
[INFO] co.elastic.apm:external-plugin-test ................ SUCCESS [  0.007 s]
[INFO] co.elastic.apm:plugin-instrumentation-target ....... SUCCESS [  0.013 s]
[INFO] co.elastic.apm:external-plugin-app ................. SUCCESS [  0.014 s]
[INFO] co.elastic.apm:external-plugin ..................... SUCCESS [  8.210 s]
[INFO] co.elastic.apm:apm-agent-attach .................... SUCCESS [  0.030 s]
[INFO] co.elastic.apm:apm-agent-attach-cli ................ SUCCESS [ 16.675 s]
[INFO] co.elastic.apm:application-server-integration-tests  SUCCESS [  1.750 s]
[INFO] co.elastic.apm:spring-boot-1-5 ..................... SUCCESS [  0.239 s]
[INFO] co.elastic.apm:spring-boot-2 ....................... SUCCESS [  0.010 s]
[INFO] co.elastic.apm:spring-boot-2-base .................. SUCCESS [  1.517 s]
[INFO] co.elastic.apm:spring-boot-2-jetty ................. SUCCESS [  0.142 s]
[INFO] co.elastic.apm:spring-boot-2-tomcat ................ SUCCESS [  0.087 s]
[INFO] co.elastic.apm:spring-boot-2-undertow .............. SUCCESS [  0.090 s]
[INFO] co.elastic.apm:runtime-attach ...................... SUCCESS [  0.006 s]
[INFO] co.elastic.apm:runtime-attach-app .................. SUCCESS [  0.040 s]
[INFO] co.elastic.apm:runtime-attach-test ................. FAILURE [  1.525 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  21:26 min
```
</details>

## Additional context

--- a/integration-tests/runtime-attach/runtime-attach-test/src/test/java/co/elastic/apm/testapp/RuntimeAttachTestIT.java
+++ b/integration-tests/runtime-attach/runtime-attach-test/src/test/java/co/elastic/apm/testapp/RuntimeAttachTestIT.java
@@ -55,7 +55,7 @@ import java.util.stream.Stream;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
 
-class RuntimeAttachTest {
+class RuntimeAttachTestIT {
 
     private static final ObjectName JMX_BEAN;
 
@@ -392,7 +392,7 @@ class RuntimeAttachTest {
             throw new IllegalStateException(e);
         }
         return found.findFirst().orElseThrow(() -> {
-            throw new IllegalStateException("Unable to find packaged test application in folder :" + folder.toAbsolutePath());
+            throw new IllegalStateException(String.format("Unable to find packaged test application in folder : %s, make sure to run 'mvn package' in folder '%s' first", folder.toAbsolutePath(), folder.toAbsolutePath().getParent()));
         });
     }
 

